uwblueprint/focus-on-nature
# Implement add, delete and edit question functionality.

## Notion ticket link

[Camp Creation/Editing: Registration Form - Accordion/Card Component & Page Layout](https://www.notion.so/uwblueprintexecs/Camp-Creation-Editing-Registration-Form-Accordion-Card-Component-Page-Layout-48f2375da8f34e69ade03390048bdd4f)
[Camp Creation/Editing: Registration Form - Add Question](https://www.notion.so/uwblueprintexecs/Camp-Creation-Editing-Registration-Form-Add-Question-fe117df2285442fa803b6c2e120642b7)
[Camp Creation/Editing: Registration Form - Edit/Delete Question](https://www.notion.so/uwblueprintexecs/Camp-Creation-Editing-Registration-Form-Edit-Delete-Question-2adf22bd8c184baca555f189fdbe1a9e)



## Implementation description
* Added accordion components/cards to display the added questions
* Linked the Add Question common modal with the registration form page
* Configured the modal to accept optional default state used in editing
* Created a delete question method



## Steps to test
1. Go to /admin/add-camp, the final page (registration form)
2. Try different functions i.e. adding a question, deleting a question, displaying different accordions, editing questions, editing different types of questions, etc.
3. Or, see the video below:

https://user-images.githubusercontent.com/30396273/202589258-cd10dd10-7e7d-41a1-b62c-39c98d9ca51b.mp4




## What should reviewers focus on?
* Design implementations (if something could visually be improved)
* Correctness (everything works)
* Code structure (whereever it makes sense to add components to common files)


## Checklist
